server_admin/topics/user-federation.adoc

```diff
@@ -27,3 +27,20 @@ image:{project_images}/user-federation.png[]
 
 On the right side, there is an `Add Provider` list box.  Choose the provider type you want to add and you will be brought to the configuration page of that provider.
 
+=== Dealing with Provider Failures
+
+If a User Storage Provider fails, i.e. if your LDAP server is down, you may have trouble logging in and may not be able to view users in the admin console.
+{project_name} does not catch failures when using a Storage Provider to lookup a user.  It will abort the invocation.  So, if you have a Storage Provider with a higher
```

Comment:
The style guide requires us to use only one space after a period at the end of a sentence.

Reply:
Style guide needs to change.  2 spaces after a period is a standard typing practice. Something I learned 30 years ago.

Reply:
Oh crap...did your changes get in?  You requested changes and I never used that github feature.

Reply:
Sorry, put in your "that is" suggestion here:

https://github.com/keycloak/keycloak-documentation/pull/283

Reply:
Sorry, put in your "that is" suggestion here:

#283

Reply:
The style guide used is not going to change. Red Hat didn't write it. The company as a whole has adopted the IBM Style Guide for all technical writing.

The single space after a period has been the standard in technical writing for over a decade. The reason is that in the era of electronic, rather than manual, typesetting we should allow our presentation layer to determine how it looks and is spaced rather than force it. Right now, I'm sure our presentation layer is compensating for double spaces to make them single, but processing speeds up a little when it has less to do.

Here's a pic of the style guide entry:
![2018-01-04 06 32 27](https://user-images.githubusercontent.com/660205/34563836-e291126c-f119-11e7-9b4e-92f1a6392f1a.jpg)